ci-operator/step-registry/cert-manager/install/catalog/cert-manager-install-catalog-commands.sh

```diff
@@ -66,7 +66,101 @@ spec:
   sourceType: grpc
   image: $INDEX_IMG
```

Comment:
Can we support the fbc and Product index too?

Reply:
Yes, any kinds of catalog image are supported